docs/use-cases.rst

```diff
@@ -26,14 +26,121 @@ Typical Use-Cases
+participants exchange information (using unicast addresses) about their publisher and subscriber entities with the rest
+of the participants, so that matching between publishers and subscribers in the same topic can occur. As the number of
+participants, publishers, and subscribers increases, the meta-traffic, as well as the number of connections, increases
+exponentially, severely affecting the setup time and memory consumption. Fast-RTPS provides extra features that expand
+the DDS standard to adapt it to wide deployment scenarios.
+
++-----------------------------------+---------------------------------------------------------------------------------+
+| Feature                           | Purpose                                                                         |
++===================================+=================================================================================+
+| Server-Client Discovery Mechanism | This feature is intended to substitute the standard SEDP and SPDP protocol with |
+|                                   | a discovery based on a server-client architecture, where all the meta-traffic   |
+|                                   | goes through a hub (server) to be distributed throughout the network            |
+|                                   | communication nodes.                                                            |
++-----------------------------------+---------------------------------------------------------------------------------+
+| Static Discovery                  | With this feature, the user can manually specify which participant should       |
+|                                   | communicate with which one and through which address and port. Furthermore, the |
+|                                   | the user can specify which publisher/subscriber matches with which one, thus    |
+|                                   | eliminating all EDP meta traffic.                                               |
++-----------------------------------+---------------------------------------------------------------------------------+
+
 .. END SUBSEC:DISCOVERY-SERVER
 
 
 .. START SUBSEC:STATIC-DISCOVERY
 
+.. _wide_deployments_static:
+
+Well Known Network Topologies
+-----------------------------
+
+It is often the case in industrial deployments, such as productions lines, that the entire network topology (hosts, IP
+addresses, etc.) is known beforehand. Such scenarios are perfect candidates for Fast-RTPS STATIC discovery mechanism,
+which drastically reduces the middleware setup time (time until all the entities are ready for information exchange),
+while at the same time limits the connections to those strictly necessary. As explained in the :ref:`discovery` section,
+all Fast-RTPS discovery mechanisms consist of two steps: Participant Discovery Phase (PDP), and Endpoint Discovery Phase
```

Comment:
You state the long version of the acronyms in the introduction. Maybe extend them here could be repetitive.